src/80211_if.c

```diff
@@ -2941,6 +2974,13 @@ static int uccp420wlan_init(void)
 		}
 	}
 
+	error=start_core(hw);
```

Comment:
Ok i understand your intention. But the split is wrong, when you send CMD_SET (prog_reset) RPU will be actively RX packets so it should be done only when user sends "ifconfig wlan0 up" we cannot by default send this after module_init.

For now i suggest we go with moving fw_loading to init and let me check with FW team how they are using bt_info and then we can take a call.